### ISSUE TITLE
Debug Level Log

### ISSUE THREAD

**Issue by mscbpi** [purestorage/pso-csi#90]
Default in values.yaml is:
```
app:
  debug: false
```

Still the container is very talky with debug level messages: 

```
pso-csi-container time="2020-09-30T11:27:49Z" level=debug msg="executing atomic update" collection=nodecollection.Node filter="id = '150ebce8-754f-4a63-8346-c9e8c8dd630c'"
--
```

**Comment by sdodsley** [comment]
@dsupure can you have a look at this?

**Comment by dsupure** [comment]
Hi @mscbpi, @sdodsley  
Yes. I am aware of it. PSO 6.x produces many more logs than 5.x due to our volume metadata service (cockroach-db). 
@mscbpi : Does the current PSO logger affect your system/environment and you need a quick solution for it?

I've created a JIRA to track this issue and we plan to address it in 20Q4. We plan to revise our current logs and build a proper mechanism to decide the message levels that users can quickly get the key messages. 
  
I'll hold off this one to Q4 and don't want to overload the team given the tasks we have in Q3.
Let me know how you think.

**Comment by mscbpi** [comment]
@dsupure 
We log our cluster logs into an Elastic instance so it's quite polluting the database. A quick fix would be appreciated.
My concern is that you have a debug switch in your chart however log level `debug` is still present.

**Comment by dsupure** [comment]
Ah, I overlook it. Let me see what happens here.

**Comment by haibinxie** [comment]
It's a bug, will make a fix.

**Comment by mscbpi** [comment]
You still have debug level log by default for container `db_deployer`.

@haibinxie

**Comment by haibinxie** [comment]
I guess this fix did not work, it should be -debug or --debug=false @dsupure FYI please reassign owner.
https://github.com/purestorage/pso-csi/commit/347f6636bbc3a0d12d08486a032040ba95fcf9d8#diff-21c41dc0aa665e39c1501a8b25dba98d83a1a1d485fe4d7bd01e35561342b0fa

**Comment by dsupure** [comment]
Hi @mscbpi : 
sorry for the inconvenience that we overlook it. I'll have @david-enli to take a look and make sure we fix it.

**Comment by david-enli** [comment]
hello @mscbpi , sry for the delay, if you want, you could make the same simple change I made in this PR: https://github.com/purestorage/pso-csi/pull/126, I've tested it on my cluster and it should work as expected. Once the team has the bandwidth to review it, it will be merged to the helm-chart, thanks!

**Comment by mscbpi** [comment]
Hi @david-enli,

not sure yet:
`time="2021-01-15T20:54:21Z" level=info msg="no nodes need to be created or decommissioned" reconciler_name=controller_intrusion request=pure-pso/pso-db`

this looks like info level debugging.

my settings:

```
clusterID: <some_clustername>
app:
  debug: false
arrays:
  FlashArrays:
    - MgmtEndPoint: "some_fa_mgmt"
      APIToken: "some_token"
  FlashBlades:
    - MgmtEndPoint: "some_fb_mgmt"
      APIToken: "<some_token>"
      NFSEndPoint: "<some_nfs_ip>"

flashblade:
  snapshotDirectoryEnabled: "false"
  exportRules: "<some_subnet>(rw,no_root_squash)"
```

**Comment by david-enli** [comment]
hi @mscbpi sry for the delayed response. Just to confirm, It seems that you have stopped seeing DEBUG level messages. We understand the inconvenience on your end. For the next release(currently aimed for Mid Feburary), the team will review PSO6 code base and make some changes to the log messages to make sure the log messages are more aligned with the standard log levels, this will hopefully make the messages less verbose for your use-case. Thank you

**Comment by mscbpi** [comment]
hi @david-enli  is it worth testing 6.1.x and compare verbosity of logs ?

**Comment by david-enli** [comment]
hi @mscbpi , hope you are doing well. Unfortunately, we have not got a chance to include this fix in 6.1.x. But we already have an engineer assigned to revisit the log messages in the containers you mentioned (db-deployer and cockroach operator) to adjust and clean up some of the messages so it won't be as verbose. We will follow up in this thread again once we have a new patch. It will be out before our next major release as a patch. Thanks!
